app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :author
   belongs_to :post
+
+  after_create :increment_likes_counter
+
+  def increment_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Nice work using the rails callback method. However, you will need the `after_save` method instead of the `after_create` method here. It is also advisable to make the `increment_likes_counter` method private. when using the rails active callbacks `after_save` method to call it. :+1: